iothub_client/tests/iothubclient_uploadtoblob_e2e/iothubclient_uploadtoblob_e2e.c

```diff
@@ -251,32 +251,34 @@ static void e2e_uploadtoblob_test(IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol, IOT
         deviceToUse = IoTHubAccount_GetSASDevice(g_iothubAcctInfo);
     }
     ASSERT_IS_NOT_NULL(deviceToUse);
-
-    IOTHUB_CLIENT_HANDLE iotHubClientHandle = IoTHubClient_CreateFromConnectionString(deviceToUse->connectionString, protocol);
-    ASSERT_IS_NOT_NULL(iotHubClientHandle, "Could not invoke IoTHubClient_CreateFromConnectionString");
-
-    if (accountAuthMethod == IOTHUB_ACCOUNT_AUTH_X509)
+    if (deviceToUse != NULL)
```

Comment:
Do we need the if (deviceToUse) check? Right above there's an ASSERT_IS_NOT_NULL(deviceToUse) so if this is NULL the test should immediately exit.

Reply:
Yes, this is flagged by SDL check. I fix them as I see them.